synapseclient/multipart_upload.py

```diff
@@ -329,18 +348,34 @@ def _multipart_upload(syn, filename, contentType, get_chunk_function, md5, fileS
     time_upload_started = time.time()
     progress=True
     retries=0
-    mp = Pool(8)
+
+    try:
+        mp = Pool(8)
+        expiredValue = Value(c_bool, False)
+        singleThreaded = False
+    except OSError:
+        mp = DummyPool()
+        expiredValue = DummyValue(False)
```

Comment:
If you do the refactor suggested above,  just revert the changes to `expired`/`expiredValue `. The only issue reverting it would cause is repeated warning messages about url expiration (but upload will still work) when you have a large file upload (should be fixed by #467)